jvm/src/main/java/com/intel/oap/substrait/expression/SelectionNode.java

```diff
@@ -17,27 +17,29 @@
 
 package com.intel.oap.substrait.expression;
 
-import io.substrait.*;
+import io.substrait.proto.*;
 
 import java.io.Serializable;
 
 public class SelectionNode implements ExpressionNode, Serializable {
     private final Integer fieldIdx;
 
     SelectionNode(Integer fieldIdx) {
-        this.fieldIdx = fieldIdx;
+        this.fieldIdx = fieldIdx + 1;
```

Comment:
Why adding one here?

Reply:
In protobuf, the value 0 means this class was reset:
![image](https://user-images.githubusercontent.com/9430290/149120896-e6c083e2-abbd-4fd0-b476-82738da8aaf3.png)
So set the start index to 1.

Function mappings are also set the start index to 1.

Reply:
Fixed: the index starts from 0.